src/_app/DefaultApp.tsx

```diff
@@ -1,7 +1,17 @@
 import React from 'react';
-import { NextApp } from '../commonTypes';
+import type { NextPage } from 'next';
+import type { PageProps } from '../commonTypes';
+import type { AppContext, AppInitialProps } from 'next/app';
 
-const DefaultApp: NextApp = function DefaultApp({ Component, pageProps }) {
+type Props = {
+  Component: NextPage;
+  pageProps: PageProps | undefined;
+};
+
+/* TODO: we should be using DefaultApp from next/app as it has some custom logic */
+const DefaultApp: React.FC<Props> & {
```

Comment:
Changed this to make this file source of truth for the type that gets imported into `commonTypes` so that we don't have to do the hack https://github.com/toomuchdesign/next-page-tester/pull/123/files#diff-9e5169db5d43285f00b985e7da04559890c20074f544434458f58f0b42084780L7 to make it available in the `dist` folder